data/map planets.txt

```diff
@@ -136,6 +136,14 @@ planet "Alta Hai"
 	bribe 0
 	security 0.9
 
+planet Alvorada
```

Comment:
Revisiting descriptions to check over for if they need edits to fit with recent chances...
I think Alvorada's still works even with going back to being an Asteroid Base. Would it need to lose the 'moon' tag though? If it's still going to orbit Baily it is serving as a moon to the planet.

Reply:
It's no longer orbiting Bailey and now orbits the stars, so yeah.